patches/bess/0012-Dpdk-concurrent-hash-support-pipeline-improvement.patch

```diff
@@ -1,15 +1,15 @@
-From 9f762576e06c194708a8f9977a1992fcf43f7a0e Mon Sep 17 00:00:00 2001
+From 82a661d06d62d80d43af3c55725539ec77c20583 Mon Sep 17 00:00:00 2001
 From: Amar Sri <amarsri28@gmail.com>
-Date: Thu, 3 Jun 2021 00:56:00 -0700
-Subject: [PATCH] Enabling concurrency & resolving issues
+Date: Thu, 10 Jun 2021 02:29:17 -0700
+Subject: [PATCH] bug #282 resolved
```

Comment:
no need to change this everytime the patch is updated.

Reply:
it ll given an idea what exactly patch set does, as each time we have to replace entire patch file, which consist lot of changes.

Reply:
The PR commit message can capture that, OR, you can add to the patch description itself.

Reply:
By git theory, This inside Subject is  updated by git upon running command when patch file created,  both i.e 'patch commit message' & patch file subject shall remain in sync to avoid any confusion, moreover someone can easily looked into git logs, relate it with previous patch chain, perform operations(revert etc).